src/Entity/Organization.php

```diff
@@ -0,0 +1,132 @@
+<?php
+declare(strict_types=1);
+
+namespace AppBundle\Entity;
+
+use Symfony\Component\Validator\Constraints as Assert;
+
+class Organization
+{
+    private $id;
+    private $name;
+    private $address;
+    private $logo;
```

Comment:
The `Organization` entity should be kept super simple (we would like to use it for other things in the future). 
Only those fields should be there.

Reply:
okay